src/planning.py

```diff
@@ -209,9 +209,17 @@ def _run_low_level_search(
                 cur_idx += 1
                 # Check atoms against expected atoms_sequence constraint.
                 assert len(traj) == len(atoms_sequence)
-                atoms = utils.abstract(traj[cur_idx], predicates)
-                if atoms == {atom for atom in atoms_sequence[cur_idx]
-                             if atom.predicate.name != _NOT_CAUSES_FAILURE}:
+                # The NSRT defines a scope: objects that are relevant for this
```

Comment:
Move this comment above the `assert len(traj) == len(atoms_sequence)` line, so that it's attached with the `# Check atoms against expected atoms_sequence constraint.` comment, otherwise that comment seems like it's only for the subsequent line.